static/adminIcons.ts

```diff
@@ -1,163 +0,0 @@
-export const ChevronLeft = `<svg id="ChevronLeft" xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" stroke="currentColor"><path stroke-linecap="round" stroke-linejoin="round" d="M15.75 19.5L8.25 12l7.5-7.5" /></svg>`;
```

Comment:
I think change folder of this files could broke something.

cc. @caroluchoa

Reply:
change because intellisense doesn't work inside static folder rsrsrs
and also because I don't think it makes sense for these files to be in that folder

Reply:
@caroluchoa Why is this there?

https://storefront-vtex.deco.site/adminIcons.ts maybe?

Reply:
no particular reason, it seems to be fine changing it

Reply:
nice

Reply:
wait, I see an error here:
<img width="521" alt="image" src="https://github.com/deco-sites/storefront/assets/43659990/cb988a4a-cd9f-4743-b686-0b3d6d847bfa">
when rendering the Button Group widget, that uses those icons

Reply:
I moved the file to the static folder

Reply:
@saymondamasio This issue is probably not in the file location, but somewhere else. Maybe something in the code is making the icons values for Button Group widget not work.

The error is still there after pulling your last changes:
<img width="581" alt="image" src="https://github.com/deco-sites/storefront/assets/43659990/171df00a-59f1-4b97-ada4-19224759a326">

Reply:
it looks like the loader is not being called, even before my changes the error persists

Reply:
probably the changes recently made on storefront main crashed something
@guitavano

Reply:
@saymondamasio this issue I pointed out was fixed on admin's last commit

Reply:
@caroluchoa show, I updated it with main

Reply:
@guitavano @caroluchoa 
I verified that the icon component is not working, and also the icon loader is blocked

![image](https://github.com/deco-sites/storefront/assets/50559834/2d24d80d-ec6e-4bc2-a607-533c5771ff82)